bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/SpProgramFactory.scala

```diff
@@ -70,13 +70,13 @@ object SpProgramFactory {
     prog.setThesis(isThesis(proposal))
 
     prog.setPIInfo(piInfo(proposal))
-    hostNgoEmail(proposal) foreach { e => prog.setPrimaryContactEmail(e) }
+    hostNgoEmail(proposal) foreach prog.setPrimaryContactEmail
```

Comment:
I'll make this change and the changes to `Proposal` from `def` to `lazy val` and push another commit momentarily. Thanks for the tips!